osbuild/util/toml.py

```diff
@@ -44,6 +44,10 @@
         _wargs = args
         break
     except ModuleNotFoundError:
+        # Workaround for tomli_w being removed from c10s - use local copy then
+        if module == "tomli_w":
+            spec = importlib.util.spec_from_file_location('tomli_w', 'tomli_w.py')
+            _tomlw = importlib.util.module_from_spec(spec)
```

Comment:
No need for all this now.  The file's part of osbuild now, so let's always use it and ignore everything else (for writing).

Reply:
So if I will remove this - it will be picked up in the normal loop iterating over modules when `tomli_w` will be processed?
I am wondering - what if both are presented then - system one and the one in sources - if the one from sources will be always 1st to pick?

Reply:
What I was trying to do here is what we discussed - try to use system library, then, if not presented - use local copy of module. Although I was never playing with importlib and import order before :]

Reply:
@mvo5 wisdom may be needed here :)

Reply:
Always use the vendored version even if something is installed on the system that provides the same.